testsuite/tests/typing-local/local.ml

```diff
@@ -994,9 +1023,9 @@ let foo x =
 Line 4, characters 2-5:
 4 |   foo ()
       ^^^
-Error: This value escapes its region.
-  Hint: This function cannot be local,
-  because it is the function in a tail call.
+Error: This value is "local"
+       but is expected to be in the parent region or "global"
+       because it is the function in a tail call.
```

Comment:
"a tail-called function" would look nicer here.

Reply:
I somewhat prefer the original phrasing - saying "it's a tail-called function" sounds like describing an inherent property of the function itself, when really we want to describe how it's being used in a particular context.

Reply:
Resolving it for now.